subprojects/docs/src/docs/userguide/running-builds/introduction/dependency_management_basics.adoc

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_basics]]
+= Dependency Management Basics
+
+Gradle has built-in support for *dependency management*.
+
+image::gradle-basic-7.png[]
+
+Dependency management is an automated technique for declaring, resolving, and using functionality required by a project.
+
+Gradle build scripts defines a process to build projects that contain some dependencies.
+Dependencies refer to JARs, plugins, libraries, or source code that support building your project.
+
+The Gradle build system makes it easy to include external binaries or other library modules to your build as dependencies.
```

Comment:
```suggestion
The Gradle build system makes it easy to include external binaries or other libraries to your build as dependencies.
```

Much as `Component` is a keyword in Gradle, so is `Module`. If we can avoid it, I think we should stay away from this word until we present a more formal definition